gulpfile.js

```diff
@@ -84,7 +84,7 @@ gulp.task('build:js', function() {
         //.pipe(sort())
         .pipe(concat('app.js'))
         .pipe(gulp.dest((BUILD_DEST + 'js')))
-        .pipe(uglify())
+        //.pipe(uglify())
```

Comment:
I had a suspicion it was borking some of my code but it seems to be working now so I'll put it back in.